### ISSUE TITLE
Binding both HTTP and HTTPS ports when running in dev

### ISSUE THREAD

**Issue by byF** [playframework/playframework#9055]
### Play Version (2.5.x / etc)
2.6.15

### API (Scala / Java / Neither / Both)
Scala

### Operating System (Ubuntu 15.10 / MacOS 10.10 / Windows 10)
macOS 10.14.3

### JDK (Oracle 1.8.0_72, OpenJDK 1.8.x, Azul Zing)
Oracle 1.8.0_181

### Library Depedencies
`play-sbt`

### Expected Behavior

I bind the ports for development in sbt with: 
```
PlayKeys.devSettings ++= Seq("play.server.http.port" -> "9090", "play.server.https.port" -> "9443")
```

Using `run` should start listeners for both HTTP and HTTPS.

### Actual Behavior
When I execute `run`

![image](https://user-images.githubusercontent.com/592682/53193485-5c284e00-3611-11e9-83da-641294906240.png)

When I execute `run -Dhttps.port=9443`

![image](https://user-images.githubusercontent.com/592682/53193599-af9a9c00-3611-11e9-8218-0ae82d2678ec.png)

**Comment by mkurz** [comment]
You are right, in dev mode this is inconsistent. I had a look at the source: prod mode should not be effected (I think).
For dev mode this line is relevant here:
https://github.com/playframework/playframework/blob/8d20cffb8a0bf25588907e55bbb62264fc6a0393/transport/server/play-server/src/main/scala/play/core/server/DevServerStart.scala#L49-L51

I will work on a fix as soon I have more time (already worked on it a bit, but nothing to commit yet).

**Comment by mkurz** [comment]
Note to myself:
Also check `Reloader.scala` because it already reads to ports and passes it to `DevServerStart`:
https://github.com/playframework/playframework/blob/67ebebab145cd8073218f077428cc79df7c5aae3/dev-mode/run-support/src/main/scala/play/runsupport/Reloader.scala#L111-L120

**Comment by mkurz** [comment]
Fixes by #9093
